aws-api/src/main/java/com/amplifyframework/api/aws/GsonListDeserializer.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
2020

Reply:
This file was actually from 2019. It was removed only for a few days.

Reply:
makes sense! Go on ahead then :)